FlightJournal.Web/Controllers/TrainingLogController.cs

```diff
@@ -128,6 +126,8 @@ public PartialViewResult GetRecentFlights(string flightId)
             foreach (var f in theFlights)
             {
                 var ae = db.AppliedExercises.Where(x => x.FlightId == f.FlightId).Where(x => x.Grading != null && x.Grading.Value > 0);
+                if (ae.IsNullOrEmpty())
```

Comment:
er det ikke mere en ren is null den er ude i ?
Empty er normalt til string checks... eller overser jeg noget

Reply:
Det var egentlig bare hvad der sad i fingrene - jeg er vant til at bruge IsNullOrEmpty på alle containere...
Og her må det være korrekt, vi vil kun fortsætte hvis der er nogle træningsdata.

Men ok, en Where() skulle ikke kunne returnere null, så !Any() ville give samme resultat.